sw/host/opentitanlib/src/tpm/access.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+pub struct TpmAccess(u8);
```

Comment:
Would this struct be better modeled with the `bitflags!` macro?

Reply:
Yes, will update.